src/sage/geometry/cone.py

```diff
@@ -6194,6 +6194,205 @@ def Z_operators_gens(self):
         """
         return [ -cp for cp in self.cross_positive_operators_gens() ]
 
+    def max_angle(self, other=None, exact=True, epsilon=0):
+        r"""
+        Return the maximal angle between ``self`` and ``other``.
+
+        The maximal angle between two closed convex cones is the
+        unique largest angle formed by any two unit-norm vectors in
+        those cones. In pathological cases, this computation can fail.
+
+        If it fails when ``exact`` is ``True`` and if each of your
+        cones :meth:`is_strictly_convex`, then a second attempt will
+        be made using inexact arithmetic. (This sometimes avoids the
+        problem [Or2024]_). If the computation fails when your cones
+        are not strictly convex or when ``exact`` is ``False``, a
+        :class:`ValueError` is raised.
+
+        INPUT:
+
+        - ``other`` -- (default: ``None``) a rational, polyhedral
+          convex cone
+
+        - ``exact`` -- (default: ``True``) whether or not to use exact
+          rational arithmetic instead of floating point computations;
+          beware that ``True`` is not guaranteed to avoid floating
+          point computations if the algorithm runs into trouble in
+          rational arithmetic
+
+        - ``epsilon`` -- (default: ``0``) the tolerance to use when
+          making comparisons
+
+        .. WARNING::
+
+          Using inexact arithmetic (``exact=False``) is faster, but
+          this computation is only known to be stable when both of
+          your cones are strictly convex (or when one of them is the
+          entire space, but the maximal angle is obviously `\pi` in
+          that case).
+
+        OUTPUT:
+
+        A triple `\left( \theta_{\text{max}}, u, v \right)`
+        containing,
```

Comment:
```suggestion
        A triple `\left( \theta_{\max}, u, v \right)`
        containing:
```

Reply:
When the subscript is a word and not a variable I prefer to see it in text face, but I've no idea if I'm being (in)consistent and don't want to waste time checking, so I've gone ahead and changed this.

Reply:
Note that I had `\max`, not `max`. So it will display as text. I agree with you that text formatting is better.

Reply:
Ah, I missed the `\` in the diff. Thanks, I'll put it back.